.gitignore

```diff
@@ -61,6 +61,11 @@ docs/_build/
 # PyBuilder
 target/
 
+# pyenv
+#   For a library or package, you might want to ignore these files since the code is
+#   intended to run in multiple environments; otherwise, check them in:
+.python-version
+
```

Comment:
I played with pdm and after `pdm use 3.9.21` (I guess so) the ".python-version" has appeared.